src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/jpa/ActivityScheduleMapping.kt

```diff
@@ -11,14 +11,16 @@ data class ActivityScheduleMapping(
   @Id
   val scheduledInstanceId: Long,
 
-  val nomisCourseScheduleId: Long,
+  var nomisCourseScheduleId: Long,
 
-  val mappingType: ActivityScheduleMappingType,
+  var mappingType: ActivityScheduleMappingType,
 
   val activityScheduleId: Long,
 
   val whenCreated: LocalDateTime? = null,
 
+  var whenUpdated: LocalDateTime? = null,
```

Comment:
interested why you haven't gone for the `@LastModifiedDate` and `@CreatedDate` annotations?